extensions/http/src/main/java/org/mule/extension/http/api/listener/builder/HttpListenerResponseBuilder.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.http.api.listener.builder;
+
+import org.mule.extension.http.api.HttpMessageBuilder;
+import org.mule.runtime.core.api.MuleEvent;
+import org.mule.runtime.extension.api.annotation.Alias;
+import org.mule.runtime.extension.api.annotation.Parameter;
+import org.mule.runtime.extension.api.annotation.param.Optional;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+import javax.activation.DataHandler;
+
+/**
+ * Component that specifies how to create a proper HTTP response.
+ *
+ * @since 4.0
+ */
+@Alias("simple-response-builder")
```

Comment:
No, it's just to standardise both listener and request side builders. These should not survive long in any case.